packages/circuit-ui/components/NotificationBanner/NotificationBanner.module.css

```
@@ -1,8 +1,7 @@
 .base {
   position: relative;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
+  display: grid;
+  grid-template-areas: "content image";
```

I think we could simplify this further by "flattening" content, image and button grid areas, and using
```suggestion
        grid-template-areas: "image content button"
```
on large devices, and 
```suggestion
grid-template-areas: "content content image" "button . image"
```
on smaller devices. That way we can get rid of the wrapper all together :)

---

Nice suggestion! I flattened the content ([commit](https://github.com/sumup-oss/circuit-ui/pull/3147/commits/0b778e193ad920e43bc47bc4fc25f70ad1e60c89)) :)